dist/pom.xml

```diff
@@ -409,6 +409,42 @@
                     </excludes>
                 </configuration>
             </plugin>
+            <plugin>
+                <!-- The dependency scopes of spark-rapids-jni and jucx are provided in sql-plugin, shuffle-plugin and udf-compiler submodules, -->
```

Comment:
NIT: 
Multiple lines comment can be achieved by 
```
<!--
spark-rapids-jni and jucx are ignored during the dedup process, because they are
 marked as "provided" dependencies and no longer exist in aggregator jars.

Now this "unpack" goal from the dependency plugin will pull out spark-rapids-jni and 
jucx directly from maven local or remote repo, and unpack them into our "parallel-world". 
Then they will be included into the dist jar automatically with packing the
 "parallel-world" folder.

Note all shims will use the same version of spark-rapids-jni and jucx.
-->
```

Reply:
done